core/node/eth_sender/src/metrics.rs

```diff
@@ -98,7 +98,7 @@ pub(super) struct EthSenderMetrics {
     /// Last L1 block observed by the Ethereum sender.
     pub last_known_l1_block: Family<BlockNumberVariant, Gauge<usize>>,
     /// Number of in-flight txs produced by the Ethereum sender.
-    pub number_of_inflight_txs: Gauge<usize>,
+    pub number_of_inflight_txs: Family<OperatorType, Gauge<usize>>,
```

Comment:
very nice! Alternatively you could use a `Family<ActionTypeLabel, ..>` but probably this one is easier to implement

Reply:
I'm not sure it's possible as PublishProof and Execute transactions share the pool of inflight txs